src/torch_ucc.cpp

```diff
@@ -622,21 +637,20 @@ ProcessGroupUCC::~ProcessGroupUCC() {
     comm->ucx_disconnect_eps(eps, &oob);
```

Comment:
@Sergei-Lebedev - sorry I missed this when resolving merge conflict (again :( ). I've fixed it and tested the PR internally and everything looks good without perf. impact. Please help fix this compile issue, and I will merge this PR soon.